server/rest/src/main/java/org/infinispan/rest/embedded/netty4/SecurityActions.java

```diff
@@ -17,7 +17,7 @@
  * @since 8.2
  * Temporary fork from RestEasy 3.1.0
  */
-final class SecurityActions {
+public final class SecurityActions {
```

Comment:
There is a good reason why these are not public. You will need to copy the package to your own.